elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -99,11 +101,13 @@ protected Elide(AuditLogger auditLogger,
             dataStore,
             dictionary,
             mapper,
-            ActivePermissionExecutor::new,
+            permissionExecutor,
             Collections.singletonList(new DefaultFilterDialect(dictionary)),
             Collections.singletonList(new DefaultFilterDialect(dictionary)),
             Pagination.MAX_PAGE_LIMIT,
-            Pagination.DEFAULT_PAGE_LIMIT
+            Pagination.DEFAULT_PAGE_LIMIT,
+            false,
+            HttpStatus.SC_NO_CONTENT
```

Comment:
Yes, I will remove this one. I will then quickly open a new PR to fix this situation with ElideSettings and RequestScope. Time to clean some stuff up!